tests/foreman/endtoend/test_ui_endtoend.py

```diff
@@ -359,7 +360,7 @@ def test_positive_puppet_install(self):
             session.nav.go_to_red_hat_repositories()
             # List of dictionary passed to enable the redhat repos
             # It selects Product->Reposet-> Repo
-            self.sync.enable_rh_repos(repos)
+            self.sync.enable_rh_repos(repos, REPO_TAB['RPMs'])
```

Comment:
Please, fix `REPO_TAB['RPMs']` on REPO_TAB['rpms']
That will fail the test otherwise